website/developer-docs/api/flow-executor.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Flow executor
+title: Flow executor (backend)
 ---
 
-A big focus of authentik is the flows system, which allows you to combine and build complex conditional processes using stages and policies. Normally, these flows are executed in the browser using the authentik inbuilt flow executor (/if/flows).
+A big focus of authentik is the flows system, which allows you to combine and build complex conditional processes using stages and policies. Normally, these flows are automatically executed in the browser using authentik's frontend [default flow executor (/if/flows)](../../docs/flow/executors/if-flow.md).
 
-However, any flow can be executed via an API from anywhere, in fact that is what the Web flow executor does. This means, you can, with a few requests, execute flows from anywhere, and integrate authentik even better.
+However, any flow can be executed via an API from anywhere, in fact that is what the backend flow executor does. This means, you can, with a few requests, execute flows from anywhere, and integrate authentik even better.
```

Comment:
Suggestion: "With a few requests you can execute flows from any device and integrate authentik more fully into your workflows."

Otherwise it reads, a bit, like, William Shatner!, when he's, monologuing!